BE/toDoListAPI/src/main/java/school/mindera/toDoListAPI/controller/CommentController.java

```diff
@@ -18,12 +18,12 @@ public CommentController(CommentService commentService) {
         this.commentService = commentService;
     }
 
-    @GetMapping("/{taskId}")
+    @GetMapping("/v1/{taskId}")
     public ResponseEntity<List<DTOComment>> getGroups(@PathVariable Integer taskId){
         return commentService.getComments(taskId);
     }
 
-    @PostMapping("/new-comment")
+    @PostMapping("/v1")
```

Comment:
Só v1? Deixa e ter o new comment?

Reply:
Yha pq e o unico post, e se nao fosse a cena das versoes, tinha ficado assim
```suggestion
    @PostMapping()
```